app/controllers/application_records_controller.rb

```diff
@@ -132,4 +130,13 @@ def given_or_all_department_ids
   def record_params
     params.permit(:position_id, :ethnicity, :gender)
   end
+
+  def modify_params
+    parameters = {}
+    parameters[:date] = if params[:date_for_later].present?
```

Comment:
Since it won't be set otherwise, for the sake of readability I might suggest leading with the conditional, then doing the assignment if the condition is true.